systemd/init

```diff
@@ -0,0 +1,49 @@
+#!/bin/busybox sh
```

Comment:
Should we have a `set -e` in here? 🤔

Reply:
Sure, I went ahead an added it.  Initially I was thinking to leave it out, and let the script get as far as it can, but I agree with you -- better to be a little more deterministic here and fail on any error.